packages/app/src/components/DBRowOverviewPanel.tsx

```diff
@@ -57,11 +61,59 @@ export function RowOverviewPanel({
     return eventAttributes;
   }, [eventAttributes, isHttpRequest]);
 
+  const exceptionValues = useMemo(() => {
+    const parsedEvents =
+      firstRow?.__hdx_events_exception_attributes ?? EMPTY_OBJ;
+    const stacktrace =
+      parsedEvents?.['exception.stacktrace'] ||
+      parsedEvents?.['exception.parsed_stacktrace'];
+
+    return [
+      {
+        stacktrace: JSON.parse(stacktrace ?? '[]'),
+        type: parsedEvents?.['exception.type'],
+        value:
+          typeof parsedEvents?.['exception.message'] !== 'string'
+            ? JSON.stringify(parsedEvents?.['exception.message'])
+            : parsedEvents?.['exception.message'],
+        mechanism: parsedEvents?.['exception.mechanism'],
+      },
+    ];
+  }, [firstRow]);
+
+  const hasException = useMemo(() => {
+    return (
+      Object.keys(firstRow?.__hdx_events_exception_attributes ?? {}).length > 0
+    );
+  }, [firstRow?.__hdx_events_exception_attributes]);
+
+  const mainContentColumn = getEventBody(source);
+  const mainContent = isString(firstRow?.['__hdx_body'])
+    ? firstRow['__hdx_body']
+    : firstRow?.['__hdx_body'] !== undefined
+      ? JSON.stringify(firstRow['__hdx_body'])
+      : undefined;
```

Comment:
if `firstRow['__hdx_body'] = null`, it will return string `null`.
```
const mainContent = isString(firstRow?.['__hdx_body'])
    ? firstRow['__hdx_body']
    : firstRow?.['__hdx_body'] !=null // so null and undefined will all pass
      ? JSON.stringify(firstRow['__hdx_body'])
      : undefined;
```

Reply:
hrmm... I think that's okay for now, if we observe that being wonky we can change behavior then